test/fixtures/config/actions.js

```diff
@@ -0,0 +1,16 @@
+const faker = require('faker');
+
+module.exports = {
+  generate() {
+    return {
+      id: faker.random.uuid(),
+      name: `${ faker.company.bsBuzz() } ${ faker.company.catchPhraseNoun() }`,
+      details: faker.lorem.sentences(),
+      due_date: faker.date.future(),
```

Comment:
I think maybe we should use `between` with moment here as due dates can be passed due.  not that, we'll want to test that with this fixture, but just having more-like-real data is probably better.